person.rb

```diff
@@ -1,7 +1,8 @@
+require './nameable'
 
-
-class Person 
+class Person < Nameable
```

Comment:
Again, class `Person` should have a function for adding rentals, and also rentals should be accessible outside the class, so consider having an attribute accessor for rentals.